### PR TITLE
feat(publish): Added subscribers settings.

### PR DESCRIPTION
**Pull request by eozan**
SD-2050

### CODE REVIEW

File: client/app/scripts/superdesk-publish/publish.js [hunk 1]
```diff
@@ -18,6 +19,32 @@
         };
 
         var service = {
+            fetchSubscribers: function(criteria) {
+                criteria = criteria || {};
+                criteria.max_results = 200;
```
Comment (amagdas) [inline client/app/scripts/superdesk-publish/publish.js]: We should use pagination here.

File: client/app/scripts/superdesk-publish/views/destination.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            <label translate>Path</label>
+            <input type="text" ng-model="destination.config.path">
+        </div>
+        <div class="field" ng-show="destination.delivery_type === 'database'">
+            <label translate>Connection string</label>
+            <input type="text" ng-model="destination.config.connection_string">
+        </div>
+        <div class="field" ng-if="actions">
+            <button type="button" class="btn btn-info" ng-repeat="action in actions" ng-click="action.fn(destination)">
+                <span translate>{{action.label}}</span>
```
Comment (plamut) [inline client/app/scripts/superdesk-publish/views/destination.html]: Use one-time binding syntax here (I assume action.label does not change).

File: client/app/scripts/superdesk-publish/views/subscribers.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        <button class="btn btn-info pull-right" ng-click="edit()">
+            <i class="icon-plus-sign icon-white"></i> <span translate>Add New Subscriber</span>
+        </button>
+    </div>
+    
+    <div class="content">
+        <ul class="pills-list provider-list">
+            <li ng-repeat="subscriber in subscribers._items" class="clearfix">
+                <div class="header">
+                    <h6>{{ subscriber.name }}</h6>
```
Comment (plamut) [inline client/app/scripts/superdesk-publish/views/subscribers.html]: One-time binding here as well? (`subscriber.name` probably does not get changed here?)

File: client/app/scripts/superdesk-publish/publish.js (hunk 1 above)
Comment (eozan) [inline client/app/scripts/superdesk-publish/publish.js]: I removed max_results (it was leftover code from previous work). Pagination (or filtering) will be added for subscribers, output channels and destinations in another PR.

**Comment by akintolga** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**